package.json

```diff
@@ -24,6 +24,7 @@
     "@ng-icons/bootstrap-icons": "^27.4.0",
     "@ng-icons/core": "^27.4.0",
     "express": "^4.19.2",
+    "primeicons": "^7.0.0",
```

Comment:
```suggestion

```
we're using @ng-icons/bootstrap-icons, no need for other icons library
https://ng-icons.github.io/ng-icons/#/browse-icons